app/authorisation.js

```diff
@@ -5,6 +5,11 @@ const Forbidden = require('./services/errors/authentication-error').Forbidden
 var assertUserAuthenticated = function (req) {
   if (isAuthenticationEnabled()) {
     if (!req.user) {
+      // Check if the request is coming from a valid user and
+      // store their last request. This is used to handle bookmark links
```

Comment:
This comment is a little misleading. This will store unauthenticated requests only.

Reply:
Updated the comment.